src/commands/codepush/lib/validation-utils.ts

```diff
@@ -3,10 +3,15 @@ import * as semver from "semver";
 import { AppCenterClient, models, clientRequest } from "../../../util/apis";
 import { DefaultApp } from "../../../util/profile";
 
+// Check if the given string is a semver-compliant version number (e.g. '1.2.3')
+export function isValidVersion(version: string): boolean {
+  return !!semver.valid(version);
```

Comment:
We should also validate for `/^\d+\.\d+$/.test(semverRange) || /^\d+$/.test(semverRange)` due to we also allow integer numbers lile 1.0, 1 etc...
also we need to move comment from `/^\d+\.\d+$/.test(semverRange) || /^\d+$/.test(semverRange)` about plain integer version number usage here

Reply:
Thanks, addressed!
https://github.com/Microsoft/appcenter-cli/pull/331/commits/b76a0af672a851edf70f8ad9e1b7891cf6be7b1f